README.rst

```diff
@@ -20,13 +20,13 @@ To install redis-py, simply:
 
 .. code-block:: bash
 
-    $ sudo pip install redis
+    $ pip install redis
```

Comment:
I think it would be good to provide an explicit mention of `virtualenv` and `venv`.  Maybe something like...

> Redis can be installed using pip similar to other Python packages.  Do not use `sudo` with `pip`.  It is usually good to work in a [virtualenv](https://virtualenv.pypa.io/en/latest/) or [venv](https://docs.python.org/3/library/venv.html) to avoid conflicts with other package managers and Python projects.  For a quick introduction see [Python Virtual Environments in Five Minutes](bit.ly/py-env).

Hopefully someday we have an overall better system for handling all this and a canonical description of how to handle installation, but for now we just have to steer people away from the things that come naturally.  Really, system pip mostly shouldn't be used at all in normal Linux setups (and Windows though it's not quite as problematic there).